src/id/mangatale/src/eu/kanade/tachiyomi/extension/id/mangatale/MangaTale.kt

```diff
@@ -2,13 +2,27 @@ package eu.kanade.tachiyomi.extension.id.mangatale
 
 import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
 import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
 import okhttp3.OkHttpClient
+import okhttp3.ResponseBody.Companion.toResponseBody
 import org.jsoup.nodes.Document
 
 class MangaTale : MangaThemesia("MangaTale", "https://mangatale.co", "id") {
 
     override val client: OkHttpClient = super.client.newBuilder()
         .rateLimit(20, 5)
+        .addInterceptor { chain ->
+            val request = chain.request()
+            val response = chain.proceed(request)
+            if (response.headers("Content-Type").contains("application/octet-stream") && response.request.url.toString().endsWith(".jpg")) {
```

Comment:
you can convert response body to image/jpeg to make sure all other image types get covered

[example](https://github.com/keiyoushi/extensions-source/blob/08f4f9a18318f4f472ab868c7aa84614ca94252b/src/all/tappytoon/src/eu/kanade/tachiyomi/extension/all/tappytoon/Tappytoon.kt#L34-L46)